sdks/python/apache_beam/runners/dataflow/dataflow_runner_test.py

```diff
@@ -698,6 +698,7 @@ def test_gbk_translation(self):
   def test_write_bigquery_translation(self):
     runner = DataflowRunner()
 
+    self.default_properties.append('--experiments=use_dataflow_bq_sink')
```

Comment:
Why not run this using the new sink ?

Reply:
this test checks that the transform is replaced appropriately.

Reply:
so we verify that the Native sink is being replaced appropriately